waspc/packages/prisma/src/format.ts

```diff
@@ -25,7 +25,7 @@ export async function formatSchema(
     const process = spawn(
       "npx",
       ["prisma", "format", "--schema", tmpPrismaSchemaFile.path],
-      {},
+      { shell: true },
```

Comment:
Maybe let's add a comment explaining why this is needed, to avoid somebody potentially removing this in the future due to not being aware (although tests should catch it, but still).